src/pdn/src/via.h

```diff
@@ -729,6 +723,13 @@ class TechViaGenerator : public ViaGenerator
 class Via
 {
  public:
+  struct ViaIndexableGetter
+  {
+    using result_type = odb::Rect;
+    odb::Rect operator()(const ViaPtr& via) const { return via->getArea(); }
```

Comment:
What is the 'area' of the via?  It seems like a bbox.

Reply:
Yeah, it's the bounding box